### PR TITLE
Tk 4 update currencies

### PR DESCRIPTION
**Pull request by thom-as-thom**

### CODE REVIEW

File: src/store/actions.ts [hunk 1]
```diff
@@ -34,3 +35,13 @@ export const deleteCrypto: Function = (Key, Cryptos) => {
     dispatch({type: DELETE_CRYPTO, payload: NewList});
   };
 };
+
+export const updateCurrencies: Function = () => {
```
Comment (tomasdematos) [inline src/store/actions.ts:39]: type error, Unnecessary use of Function type, you should use instead (): returnType =>

the arrow function is alredy  a function, replace this in all files

File: src/store/reducers.ts [hunk 2]
```diff
@@ -19,6 +15,29 @@ export default (state = initialState, action) => {
       ...state,
       addedCryptos: action.payload,
     };
+  } else if (action.type === UPDATE_CURRENCIES) {
```
Comment (tomasdematos) [inline src/store/reducers.ts:18]: use switch statement instead of else if

File: src/screens/AddCrypto/index.tsx [hunk 3]
```diff
@@ -1,15 +1,14 @@
...
 import {useState} from 'react';
 import {addCrypto} from '../../store/actions';
 import theme from '../../theme';
-import {IRootState} from '../../store';
+import {useAppDispatch, useAppSelector} from '../../hooks/hooks';
 
 export const AddCurrency = ({navigation}): JSX.Element => {
-  const dispatch = useDispatch();
-  const {addedCryptos} = useSelector((state: IRootState) => state.addedCryptos);
+  const dispatch = useAppDispatch();
```
Comment (tomasdematos) [inline src/screens/AddCrypto/index.tsx:10]: why you don't use useDispatch directli?

File: src/screens/AddCrypto/index.tsx [hunk 4]
```diff
@@ -1,15 +1,14 @@
...
+  const {addedCryptos} = useAppSelector(state => state.addedCryptos);
```
Comment (tomasdematos) [inline src/screens/AddCrypto/index.tsx:11]: same question

File: src/screens/AddCrypto/index.tsx (hunk 3 above)
Comment (thom-as-thom) [reply src/screens/AddCrypto/index.tsx:10]: To make sure dispatch is typed correctly. I read about it in redux documentation:
Screenshot_20220919-092111

**Review by tomasdematos — APPROVED** [review]

**Pull request merged**